docs-src/content/getting-started/_index.md

```diff
@@ -1,6 +1,10 @@
 +++
-title = "Getting started"
+title = "Getting Started"
 description = ""
 weight = 1
-alwaysopen = true
 +++
+
+This section is meant to get you started using Cubic Server.  It includes:
+
+* [a page](./quickstart) on using our web demo or our command-line interface, and
```

Comment:
At the CoW, Catherine said she thought the quick start pointing to the cli is problematic for customers, who then assume that if the cli can't do it, we can't.  She suggested removing the link to the quick start altogether.